examples/Buttons/Button.md

```diff
@@ -71,3 +71,45 @@ import { VARIANT } from 'mark-one';
   Click Me!
 </Button>
 ```
+
+### With the `forwardRef`
+Ref example: The optional `forwardRef` property is set. When the primary themed button is clicked, the focus shifts to the borderless button.
+```jsx
+import {
+  useState,
+  useRef,
+} from 'react';
+import { VARIANT } from 'mark-one';
+
+const ButtonRefExample = () => {
+  const ref = useRef(null);
+  const onButtonClick = () => {
+    setTimeout(() => ref.current.focus());
+  }
```

Comment:
Again why not `const onButtonClick = () => {ref.current.focus()}`?

Reply:
Thanks for catching this! I removed it through 8721dbb.